src/app/connections/list/list.component.html

```diff
@@ -1,47 +1,8 @@
 <ipaas-loading [loading]="loading">
-  <div class="container-cards-pf">
-    <div class="row row-cards-pf">
-      <div class="col-xs-12 col-sm-3 col-md-2 card" *ngFor="let connection of connections">
-        <div class="card-pf card-pf-view card-pf-view-xs">
-          <div class="card-pf-body">
-
-            <!-- Card Dropdown -->
-
-            <div class="card-pf-heading-kebab">
-              <div class="dropdown pull-right dropdown-kebab-pf card-pf-time-frame-filter">
-                <button type="button" class="btn btn-link dropdown-toggle" data-toggle="dropdown">
-                  <span class="fa fa-ellipsis-v"></span>
-                </button>
-                <ul class="dropdown-menu dropdown-menu-right" role="menu">
-                  <li>
-                    <a [routerLink]="[connection.id]">View</a>
-                  </li>
-                  <li>
-                    <a [routerLink]="[connection.id, 'edit']">Edit</a>
-                  </li>
-                  <li>
-                    <a>Duplicate</a>
-                  </li>
-                  <li>
-                    <a>Delete</a>
-                  </li>
-                </ul>
-              </div>
-
-              <!-- Card Title & Icon -->
-
-              <h2 class="card-pf-title">
-                <span class="fa {{ connection.icon }}"></span>
-                <a [routerLink]="[connection.id]" class="card-title">{{ connection.name }}</a>
-              </h2>
-            </div>
-
-            <!-- Card Description -->
-
-            <p>{{ connection.description | truncate : truncateLimit : truncateTrail }}</p>
-          </div>
-        </div>
-      </div>
-    </div>
+  <div class='connections list'>
+    <ipaas-connections-list-toolbar [childConnections]="connections | async"
+                                    (change)="updateConnections($event)"></ipaas-connections-list-toolbar>
+    <ipaas-connections-list-page [connectionList]="connections"
+                                 [loading]="loading"></ipaas-connections-list-page>
```

Comment:
So in the integration editor we've two pages that effectively re-use the connection list + toolbar that the user will use to select the start and end connection.  Wait, should I be saying `iPAAS Connector` here?

In any case, I only want to use the list component and the toolbar component, and I want my wrapper page to load the data that those two components look at.  I also am planning on adding an `@Output` to that list component so that I can let the user click on a connection to select it.